src/mpyl/schema/project.schema.yml

```diff
@@ -32,6 +32,7 @@ properties:
         - Energy Trading
         - Big Data
         - Flex IoT
```

Comment:
Does `Flex IoT` still exist, or should it be removed?

Reply:
Both `Big Data` and `Flex IoT` can be removed